packages/manager/modules/anthos/src/dashboard/routing.js

```diff
@@ -3,10 +3,33 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '/:serviceName',
     component: 'anthosDashboard',
     resolve: {
+      alertId: () => 'anthos_dashboard',
       currentActiveLink: /* @ngInject */ ($transition$, $state) => () =>
         $state.href($state.current.name, $transition$.params()),
       dashboardLink: /* @ngInject */ ($state, serviceName) =>
         $state.href('anthos.dashboard', { serviceName }),
+      goToState: ($state, Alerter, alertId) => (
+        state,
+        stateParams = {},
+        message = false,
+        type = 'done',
+      ) => {
```

Comment:
I'm not sure what the purpose of this is but controllers shouldn't have to worry to which state they go, they should just `goBack` or `goToDashboard` for example

Reply:
This is a kind of a reusable method, which avoids code replication in many places where we write the `goTo*` functions